packages/create-gatsby/src/init-starter.ts

```diff
@@ -1,21 +1,35 @@
 import { execSync } from "child_process"
-import execa from "execa"
+import execa, { Options } from "execa"
 import fs from "fs-extra"
 import path from "path"
 import { reporter } from "./reporter"
 import { spin } from "tiny-spin"
 import { getConfigStore } from "./get-config-store"
 type PackageManager = "yarn" | "npm"
+import c from "ansi-colors"
 
-const packageMangerConfigKey = `cli.packageManager`
+const packageManagerConfigKey = `cli.packageManager`
+
+const kebabify = (str: string): string =>
+  str
+    .replace(/([a-z])([A-Z])/g, `$1-$2`)
+    .replace(/[^a-zA-Z]+/g, `-`)
+    .toLowerCase()
 
 export const getPackageManager = (): PackageManager =>
-  getConfigStore().get(packageMangerConfigKey)
+  getConfigStore().get(packageManagerConfigKey)
 
 export const setPackageManager = (packageManager: PackageManager): void => {
-  getConfigStore().set(packageMangerConfigKey, packageManager)
+  getConfigStore().set(packageManagerConfigKey, packageManager)
 }
 
+export const clearLine = (count = 1): Promise<boolean> =>
+  new Promise(resolve => {
+    process.stderr.moveCursor(0, -count, () => {
+      process.stderr.write(`\u001b[2K`)
+      resolve()
+    })
+  })
```

Comment:
I assume you copied this from somewhere, could you paste the link here? Does this work on windows?

Reply:
Nope, all my own work. I'm assuming it works on Windows, because the other ansi sequences do, but it will need checking.

Reply:
I've added comments and a link to a list of escape codes

Reply:
Tested and confirmed working on Windows. (Try `npm init gatsby@create-gatsby`)